supabase/config.toml

```diff
@@ -150,6 +150,9 @@ url = ""
 # If enabled, the nonce check will be skipped. Required for local sign in with Google auth.
 skip_nonce_check = false
 
+[edge_runtime]
```

Comment:
@camelPhonso: 
Sure! I didn't do thorough research on that. I deleted the old seed.sql file, manually created new test data, and dumped it. So this row just appeared. I didn't modify the file because it was already small.

As for renaming, I'll do that soon.

Reply:
ah got, thanks @eagauss I thought it might have been a planned change for any other automation.

I've pinged the rest of the enbloc team about it because the documentation does make it look relevant so I think leaving it might be fine since it was probably triggered by the data changes you made anyway